src/Files.App/ViewModels/Settings/ActionsViewModel.cs

```diff
@@ -0,0 +1,224 @@
+﻿// Copyright (c) 2024 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+using System.Windows.Input;
+
+namespace Files.App.ViewModels.Settings
+{
+	public class ActionsViewModel : ObservableObject
```

Comment:
This class can be `sealed`

Reply:
Thanks for the review!
I'll make it sealed.

Reply:
This is resolved in 5504842c0ce320c4868a6beb11d81a0ff6a3ad8d